.env.example

```diff
@@ -0,0 +1,8 @@
+LOCAL_URL="http://localhost:8545"
+LOCAL_MNEMONIC="test test test test test test test test test test test junk"
+TESTNET_URL="https://api.s0.b.hmny.io"
+TEST_MNEMONIC=""
+MAINNET_URL="https://api.s0.t.hmny.io/"
+S1_URL="https://api.s1.t.hmny.io/"
+MNEMONIC=""
+TLD = "country"
```

Comment:
bad formatting

Reply:
actually this file should be removed

Reply:
Agree only need one file, I think we have used env.example in other repos like ens-app-v3 and .1.country